README.md

```diff
@@ -1,108 +1,107 @@
 # Bridge TensorFlow* to run on Intel® nGraph™ backends
 
 This directory contains the code needed to build a TensorFlow 
-plugin that can be used with Intel® nGraph™. nGraph is an [open-source C++ library, compiler and runtime] that provides developers with the means to 
-train and run DNN models efficiently on custom backends: GPU, CPU, and custom 
-silicon.
+plugin to the Intel® nGraph™ Compiler. As part of a TensorFlow 
+toolchain, nGraph can speed up training workloads using CPU; or 
+the nGraph Library and runtime suite can be used to customize 
+and deploy Deep Learning inference models that will "just work" 
+with variety of nGraph-enabled backends: CPU, GPU, and custom 
+silicon.  
 
-## Linux instructions
+*   [Linux instructions](#linux-instructions)
+*   [OS X instructions](#os-x-instructions)
+*   [Debugging](#debugging)
+*   [Support](#support)
+*   [About nGraph](#about-ngraph)
 
-There are a few ways you can run nGraph with TensorFlow. They are described in the 
-following section.
 
-## Option 1: Build nGraph bridge from source using existing TensorFlow installation
 
-1. Install TensorFlow using the instructions from the TensorFlow web site
-   https://www.tensorflow.org/install/install_linux
+## Linux instructions
 
-2. Now clone the `ngraph-tf` repo and go to the `ngraph-tf` directory
+You can connect nGraph to an existing TensorFlow installation if 
+you're running TensorFlow v1.11.0-rc1 or greater. For versions 
+released previous to that, we recommend starting with a clean 
+environment and re-building with the instructions in Option 2.
```

Comment:
Please add the following:

If you select option 1 or 2 below, then please add "import ngraph" in your python scripts right after the line that imports tensorflow e.g., "import tensorflow as tf".

If you select option 3 below, then no changes in the python script is needed. However, the version of nGraph may be a bit older in the TensorFlow repository.

Reply:
see earlier comment